mne_neuron/dipole.py

```diff
@@ -14,22 +16,29 @@ def _hammfilt(x, winsz):
     return convolve(x, win, 'same')
 
 
-def simulate_dipole(net):
-    """Simulate a dipole given the experiment parameters.
+def _clone_and_simulate(params, trial_idx):
+    from .network import Network
 
-    Parameters
-    ----------
-    net : Network object
-        The Network object specifying how cells are
-        connected.
+    if trial_idx != 0:
+        params['prng_*'] = trial_idx
```

Comment:
@blakecaldwell does this seem legit to you? For simulating different trials, I just changed the random seed. The way the random seed is changed is different (and simpler) than in HNN -- so the results could be slightly different for n_trials > 1. But is there anything else that needs to change from one trial to the next?

Feel free to fetch this branch locally and try it. I'll wait for your feedback before merging it.

Reply:
These seem like unnecessary changes at this point. Simplifying schemes like the randomization of inputs is appealing to me, but I pragmatically believe that all code cleanup changes should not change the model output.

In PR #55 (which conflicts with this PR) I have already simplified this as far as possible without changing results.
https://github.com/jasmainak/mne-neuron/pull/55/commits/9402e2c01a164422a71a20fc3d90ac0424164d4d#diff-f4f8654c734567d73fec96e0e0dacce5R112

Reply:
sure, although I don't quite understand what is different in the HNN code. It seems to be essentially setting a bunch of seeds -- unless I am missing something? Now you could argue that the particular seeds used by the HNN code are important so that we can compare things. In that case, I think we should set up a test first. Save data with 2 trials from HNN and compare against mne-neuron code in the test. And then make changes so that the test passes. That way, I am not afraid of making changes which will potentially break things ...

Reply:
I fully support a test to compare results from multiple trials. However, creating good tests is time-consuming. Look for a test in the near future as I update the PR.

Reply:
sure, if you can save the test data for 2 trials with the default parameters and send it to me, I'll add that as a test. For now, I merged since it does not break anything for single trial.